### PR TITLE
fix(dashmate): missing default values in IP and ports form

### PR DESCRIPTION
**Pull request by pshenmic**
## Issue being fixed or feature implemented

During registration with Dash Core Wallet, the default values for IP and port form are missing.  Registration with DMT is broken.

## What was done?

- Fixed default values for IP and ports form
- Fixed registration with DMT
- Fixed dashmate status test

## How Has This Been Tested?

None

## Breaking Changes

No

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added or updated relevant unit/integration/functional/e2e tests
- [x] I have made corresponding changes to the documentation

**For repository code-owners and collaborators only**
- [x] I have assigned this pull request to a milestone

### CODE REVIEW

File: packages/dashmate/src/listr/prompts/createIpAndPortsForm.js [hunk 1]
```diff
@@ -59,19 +55,15 @@ function createIpAndPortsFormFactory(defaultConfigs) {
...
-    let initialIp;
-    if (options.initialIp === null || options.initialIp === undefined) {
-      initialIp = await Promise.race([
-        publicIp.v4().catch(() => ''),
-        // Resolve in 10 seconds if public IP is not available
-        wait(10000).then(() => ''),
-      ]);
+    let { initialIp } = options;
+
+    if (!initialIp) {
```
Comment (shumkov) [inline packages/dashmate/src/listr/prompts/createIpAndPortsForm.js:60]: We should distinguish null/undefined and empty string

File: packages/dashmate/src/listr/tasks/setup/regular/registerMasternode/registerMasternodeWithCoreWallet.js [hunk 2]
```diff
@@ -209,6 +212,18 @@ function registerMasternodeWithCoreWalletFactory(createIpAndPortsForm) {
 
       state = await task.prompt(prompts);
 
+      const portStatus = await providers.mnowatch.checkPortStatus(state.ipAndPorts.coreP2PPort);
+
+      if (portStatus !== PortStatusEnum.OPEN) {
```
Comment (shumkov) [inline packages/dashmate/src/listr/tasks/setup/regular/registerMasternode/registerMasternodeWithCoreWallet.js:217]: I believe we should do it as part of form validation and not accept invalid port

**Review by shumkov — CHANGES_REQUESTED** [review]

File: packages/dashmate/src/listr/tasks/setup/regular/registerMasternode/registerMasternodeWithCoreWallet.js (hunk 2 above)
Comment (pshenmic) [reply packages/dashmate/src/listr/tasks/setup/regular/registerMasternode/registerMasternodeWithCoreWallet.js:217]: The port is itself valid, it is just an additional check and reminder for masternode operators

File: packages/dashmate/src/listr/prompts/createIpAndPortsForm.js (hunk 1 above)
Comment (pshenmic) [reply packages/dashmate/src/listr/prompts/createIpAndPortsForm.js:60]: Why? I thought it should detect in any cases (because user can easily put his own)

**Review by shumkov — APPROVED** [review]
👍

**Pull request merged**